newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -95,16 +98,24 @@ const ExtensionInstallDialog = ({
       if (canInstallExtension) {
         if (alreadyInstalled) {
           const answer = Window.showConfirmDialog(
-            'This extension is already in your project, this will install the latest version. You may have to do some adaptations to make sure your game still works. Do you want to continue?'
+            (extensionUpdate && extensionUpdate.type) === 'major'
+              ? i18n._(
+                  t`This extension update contains a breaking change. You will have to do some adaptations to make sure your game still works. We advise to back up your game before proceeding. Do you want to continue?`
+                )
+              : (extensionUpdate && extensionUpdate.type) === 'unknown'
+              ? i18n._(
+                  t`The latest version will be installed, but it isn't indicated whether this update includes breaking changes. You may have to do some adaptations to make sure your game still works. We advise to back up your game before proceeding. Do you want to continue?`
+                )
+              : i18n._(
+                  t`The latest version of the extension is about to be installed. While the extension notifies that there should be no breaking changes, be aware that any modifications you might have made to the extension since installing it will be discarded.  Do you want to continue?`
```

Comment:
> While the extension notifies that there should be no breaking changes, be aware that any modifications you might have made to the extension since installing it will be discarded.

It's hard to see what is the logic behind this.
Does it warns against possible behavior changes that reviews could have missed?
Does it imply that the breaking changes could come from the loss of user-made changes?

What seems important here is that user-made changes might be lost. The message seems too long and will probably not be read.

```suggestion
                  t`Be aware that any modifications you might have made to the extension since installing it will be discarded. Do you want to continue?`
```